client/webserver/http.go

```diff
@@ -190,6 +195,125 @@ func (s *WebServer) handleOrders(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// handleExportOrders is the handler for the /orders/export page request.
+func (s *WebServer) handleExportOrders(w http.ResponseWriter, r *http.Request) {
+	wf, ok := w.(http.Flusher)
+	if !ok {
+		http.Error(w, "unable to flush streamed data", http.StatusBadRequest)
+		return
+	}
+
+	filter := new(core.OrderFilter)
+	err := r.ParseForm()
+	if err != nil {
+		log.Errorf("error parsing form for export order: %v", err)
+		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		return
+	}
+
+	filter.Hosts = r.Form["hosts"]
+	assets := r.Form["assets"]
+	filter.Assets = make([]uint32, len(assets))
+	for k, assetStrID := range assets {
+		assetNumID, err := strconv.Atoi(assetStrID)
```

Comment:
Maybe https://golang.org/pkg/strconv/#ParseUint is slightly better in this case and for the status id, since they shouldn't be negative.

Reply:
`strconv.ParseUint(assetStrID, 10, 32)` Is probably better for the uint32, and order status is a uint16, so `strconv.ParseUint(assetStrID, 10, 16)`. I actually didn't know 0 will do base 10, but looks ok trying it out in the playground.

Reply:
0 will do base 10 unless the prefix in the string matches another base (0x for base 16). These should always be 10 though, so I've updated it to 10.

Reply:
Good to know! 0 was probably fine then, but probably don't expect hex for these smallish numbers.